src/Application/Application.ts

```diff
@@ -250,27 +250,41 @@ export class Application {
   }
 
   private getIqRequestService(args: any): IqRequestService {
-    if (args.user && args.password && args.server) {
```

Comment:
Since we set defaults here, had to modify the approach a tiny bit so that it would pick up non default values.

Reply:
so the config values work now, but the command line isn't overwriting the config: 

ajurgenson@ArtieSonaDell:~/auditjs$ npm run start -- iq -a auditjs-iq-ts -p asdf -u asdf

> auditjs@4.0.1-alpha.7 start /home/ajurgenson/auditjs
> node ./bin/index.js "iq" "-a" "auditjs-iq-ts" "-p" "asdf" "-u" "asdf"

✔ Starting application
✔ Getting coordinates for Nexus IQ Server
✔ Auditing your application with Nexus IQ Server
✔ Getting your internal application ID
✔ Submitting your dependencies to Nexus IQ Server
✔ Checking for results (this could take a minute)
Wonderbar! No policy violations reported for this audit!
Report URL: http://localhost:8070/ui/links/application/auditjs-iq-ts/report/8dfa828ad246414e82e2eaefd68921c7
ajurgenson@ArtieSonaDell:~/auditjs$

Reply:
should be failing out here

Reply:
Try: 6849142

Reply:
alright it works now, i still don't like how it errors out with the wrong password: 

ajurgenson@ArtieSonaDell:~/auditjs$ npm run start -- iq -a auditjs-iq-ts -p asdf

> auditjs@4.0.1-alpha.7 start /home/ajurgenson/auditjs
> node ./bin/index.js "iq" "-a" "auditjs-iq-ts" "-p" "asdf"

✔ Starting application
✔ Getting coordinates for Nexus IQ Server
✔ Auditing your application with Nexus IQ Server
✖ Getting your internal application ID
{ '0':
   { title: '',
     stack:
      'Error\n    at IqRequestService.<anonymous> (/home/ajurgenson/auditjs/bin/Services/IqRequestService.js:55:23)\n    at Generator.next (<anonymous>)\n    at fulfilled (/home/ajurgenson/auditjs/bin/Services/IqRequestService.js:5:58)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' },
  level: 'error',
  message:
   'There was an issue auditing your application with Nexus IQ Server',
  label: 'AuditJS',
  timestamp: '2020-01-08T21:13:48.104Z' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! auditjs@4.0.1-alpha.7 start: `node ./bin/index.js "iq" "-a" "auditjs-iq-ts" "-p" "asdf"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the auditjs@4.0.1-alpha.7 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ajurgenson/.npm/_logs/2020-01-08T21_13_48_128Z-debug.log

but i figure we can go through and do error handling for input validation all at once later, after it functions with expected values

Reply:
Agree there!